tests/http/config/batching-nested-expr.graphql

```diff
@@ -0,0 +1,42 @@
+schema @server @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: true, batch: {delay: 10}) {
+  query: Query
+}
+
+type Query {
+  posts: [Post] @http(path: "/posts")
+}
+
+type Post {
+  id: Int
+  title: String
+  body: String
+  userId: Int!
+  user: User
+    @expr(
+      body: {
+        if: {
+          cond: {const: {data: true}}
+          then: {http: {path: "/users", query: [{key: "id", value: "{{value.userId}}"}], groupBy: ["id"]}}
```

Comment:
this was panicking at https://github.com/tailcallhq/tailcall/blob/7a82e6bc7d764180f6b2f8807b6242bd52beabf6/src/lambda/io.rs#L223

Reply:
thanks!